openskistats/bearing.py

```diff
@@ -315,3 +315,100 @@ def get_bearing_summary_stats(
         bearing_magnitude_net=round(net_magnitude, 7),
         bearing_magnitude_cum=round(cum_magnitude, 7),
     )
+
+
+# def get_solar_intensity(
+#     latitudes: list[float] | npt.NDArray[np.float64]
+# ) -> pl.DataFrame:
+    
+
+HOURS_IN_DAY = 24
+HOURS_START = 9  # Start time (9 AM)
+HOURS_END = 16   # End time (4 PM)
```

Comment:
Did you evaluate existing libraries like:

- https://github.com/pvlib/pvlib-python
- https://github.com/climlab/climlab

For whether they could do these calculations and were suitable dependencies? For calculations like these that are highly technical, I think it's best if we start with existing tooling and only switch to a homegrown implementation if third party libs don't do what we want.

Code looks nice and methods appear correct to me, but also I know very little here and there are no references provided for which we can verify the methods.